dispatcher/dispatcher.go

```diff
@@ -44,9 +48,13 @@ func (cfg Config) Start(q JobQueue, m *model.Queue) Dispatcher {
 		bufferSize = m.MaxWorkers
 	}
 
+	pollingInterval := m.PollingInterval
+	if m.MaxDispatchesPerSecond != 0 {
+		pollingInterval = defaultMinPollingInterval
+	}
```

Comment:
When MaxDispatchedPerSecond is not empty, polling interval is set to default minimum value because a job flow is controlled by throttling.